src/meson.build

```diff
@@ -56,5 +56,6 @@ shared_library(meson.project_name().to_lower(),
     link_with: [keyboard_input_x11_lib],
     include_directories : vkBasalt_include_path,
     dependencies : [x11_dep, reshade_dep],
+    gnu_symbol_visibility: 'hidden'
```

Comment:
By default GNU exports all symbols (one of the rare cases MSVC actually has the better default):
https://mesonbuild.com/Release-notes-for-0-48-0.html#keyword-argument-for-gnu-symbol-visibility

Reply:
The line is missing a `,` at the end creating a meson build error:

```
../vkBasalt-0.3.2.8/src/meson.build:60:4: ERROR: Expecting rparen got id.
    install : true,
    ^
For a block that started at 54,14
shared_library(meson.project_name().to_lower(),
```

Reply:
Fixed.